### DISCUSSION TITLE
Copilot activation failed

### DISCUSSION THREAD

**Discussion by aazsamir** [community/community#15041] [Copilot Conversations] [answered]
I'm using VSCode from Snap Store on manjaro linux. I'm not sure, but I think since VSCode 1.66 update, I cannot get acces to copilot (which was working fine before), tried both Github Copilot and Nightly edition. I have windows 10 on dual boot, and it is working fine there.
That's my output from Github Authentication tab:

> [Info  - 13:31:41.68] Reading sessions from keychain...
> [Info  - 13:31:41.90] Getting sessions for user:email...
> [Info  - 13:31:41.102] Getting sessions for all scopes...
> [Info  - 13:31:41.106] Getting sessions for read:user,repo,user:email...
> [Error  - 13:31:46.330] Getting token failed: Error: Input is not valid base64-encoded data.
> [Info  - 13:31:46.330] Got 0 sessions for user:email...
> [Info  - 13:31:46.330] Got 0 sessions for ...
> [Info  - 13:31:46.330] Got 0 sessions for read:user,repo,user:email...
> [Info  - 13:31:46.566] Getting sessions for all scopes...
> [Info  - 13:31:46.566] Got 0 sessions for ...
> [Info  - 13:31:46.582] Getting sessions for read:user...
> [Info  - 13:31:46.582] Got 0 sessions for read:user...
> [Info  - 13:31:46.984] Getting sessions for read:user,repo,user:email...
> [Info  - 13:31:46.984] Got 0 sessions for read:user,repo,user:email...
> [Info  - 13:31:50.369] Getting sessions for read:user...
> [Info  - 13:31:50.369] Got 0 sessions for read:user...
> [Info  - 13:31:51.701] Logging in for the following scopes: read:user
> 

And output from Github copilot tab

> [ERROR] [default] [2022-04-16T13:32:51.741Z] GitHub Copilot could not connect to server. Extension activation failed: "Cancelled"

VSCode version

> Version: 1.66.2
> Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
> Date: 2022-04-11T07:49:24.808Z
> Electron: 17.2.0
> Chromium: 98.0.4758.109
> Node.js: 16.13.0
> V8: 9.8.177.11-electron.0
> OS: Linux x64 5.17.1-3-MANJARO snap

**Comment by FilipKufalov** [comment]
Yesterday i tried Copilot on my Laptop VSCode and I got the same message "Copilot activation failed" and I thought the beta review program is available only for one device after good 5 minutes I notice that my Laptop is not connected to internet

**Comment by aazsamir** [comment, marked as answer]
If anyone struggles with that too, try VSCode insiders version, related to that:
https://github.com/microsoft/vscode/issues/147601
